src/rubrix/server/tasks/token_classification/api/model.py

```diff
@@ -172,10 +175,9 @@ def annotated_as(self) -> List[str]:
 
     @property
     def scores(self) -> List[float]:
-        """Values of prediction scores"""
         if not self.prediction:
             return []
-        return [self.prediction.score]
+        return [ent.score for ent in self._predicted_entities()]
```

Comment:
Here, we lost record level scores information.

- Could we have both score levels, at record and entity? 
- Could we use both in same exploration?
- Should use entities scoring in conjuntion with its labels for filtering?  (Give me records with entities labelled as `?` and a low score (under 0.5))